packages/l10n/time-zone-information.js

```diff
@@ -1,14 +1,18 @@
 import PropTypes from 'prop-types';
 import createL10NInjector from './create-l10n-injector';
-import getSupportedLanguage from './utils/get-supported-language';
-import extractLanguageFromLocale from './utils/extract-language-from-locale';
+import { getSupportedLocale, mapLocaleToIntlLocale } from './utils';
 
-const getImportChunk = lang => {
-  switch (lang) {
+const getImportChunk = locale => {
+  const intlLocale = mapLocaleToIntlLocale(locale);
+  switch (intlLocale) {
     case 'de':
       return import(/* webpackChunkName: "timezone-data-de" */ './data/time-zones/de.json');
     case 'es':
       return import(/* webpackChunkName: "timezone-data-es" */ './data/time-zones/es.json');
+    case 'fr-FR':
+      return import(/* webpackChunkName: "timezone-data-fr-FR" */ './data/time-zones/fr-FR.json');
+    case 'zh-CN':
+      return import(/* webpackChunkName: "timezone-data-zh-CN" */ './data/time-zones/zh-CN.json');
```

Comment:
Eventually all apps would have to adjust their `message.js` too, correct?

Reply:
Depends. Our apps yes, custom apps probably not (that's why I also kept playground and the starter app without)